src/Carter/CarterOptions.cs

```diff
@@ -56,6 +56,13 @@ public class OpenApiOptions
         /// Determines if OpenAPI should be mapped and enabled. True by default.
         /// </summary>
         public bool Enabled { get; set; } = true;
+        
+        /// <summary>
+        /// Attribute used to ignore properties from the OpenAPI schema
+        ///
+        /// Note: Set this to null to disable.
+        /// </summary>
+        public Type SchemaIgnoreAttribute { get; set; } = typeof(System.Text.Json.Serialization.JsonIgnoreAttribute);
```

Comment:
Ahh, I see it's possible to override. Ignore the last comment :)

Reply:
Yeah didn't want to tie it to just the internal .net attribute.  Too many serialisers out there :)